### PR TITLE
Remove extra word.

### PR DESCRIPTION
**Pull request by gosharplite**

### CODE REVIEW

File: Documentation/deploy-master-single.md [hunk 1]
```diff
@@ -336,7 +336,7 @@ Ensure that the kublet will start after a reboot:
 $ sudo systemctl enable kubelet
 ```
 
-Our Pods should now we starting up and downloading their containers. To check the download progress, you can run `docker ps`.
+Our Pods should now starting up and downloading their containers. To check the download progress, you can run `docker ps`.
```
Comment (bcwaldon) [inline Documentation/deploy-master-single.md]: I believe the `we` was intended to be a `be`. Now the tense of `starting` and `downloading` are off. You should choose one of the following:

```Our Pods should now be starting up and downloading their containers.```

```Our Pods should now start up and download their containers.```

**Comment by aaronlevy** [comment]
@gosharplite this needs to be rebased before it can be merged

**Comment by gosharplite** [comment]
@aaronlevy Is there a way to do this in the web page? Or I have to figure out how to use command line to do this.

**Comment by aaronlevy** [comment]
AFAIK there isn't a way to do it on the web page.

This same typo ended up fixed in: https://github.com/coreos/coreos-kubernetes/pull/64

**Pull request closed without merging**